readhoney/routes/users.js

```diff
@@ -38,9 +39,12 @@ router.post ( '/signup', (request, response, next) => {
 
 
 router.get('/logout', (request, response) => {
-  request.logout()
-  response.redirect('/')
+    request.logout();
+    delete request.session;
```

Comment:
This is interesting. I don't know what this does, teach me? 😄 (I know that it deletes the key session from the request object, but that would only do it within the context of this request.  I suspect that the `logout` function does something like this in order to ensure the session is destroyed, but would love to learn about why/how this works.)